seedlet/inc/wpcom.php

```diff
@@ -112,3 +252,12 @@ function seedlet_wpcom_editor_scripts() {
 	wp_enqueue_style( 'seedlet-wpcom-editor-style', get_template_directory_uri() . '/inc/wpcom-style-editor.css', array(), '20200629' );
 }
 add_action( 'enqueue_block_editor_assets', 'seedlet_wpcom_editor_scripts' );
+
+/**
+ * Enqueue CSS and JS for customizer pane.
+ */
+function seedlet_enqueue_message_scripts() {
+	wp_enqueue_script( 'seedlet-customize-message-wpcom-script', get_template_directory_uri() . '/inc/wpcom-customize-message.js', array( 'customize-controls' ), wp_get_theme()->get( 'Version' ) );
```

Comment:
We can remove `'seedlet-customize-message-wpcom-script'` and its related JS file `/inc/wpcom-customize-message.js` entirely. 

The color a11y warning message is added with PHP (see below) and so we only need to keep the `'seedlet-customize-message-wpcom-style'` CSS to style the message. 

https://github.com/Automattic/themes/blob/b2b6229bf897596c11b35900c02bcb0ab90937dc/seedlet/inc/wpcom.php#L157-L169

Reply:
Thank you! Should be all set as of 7cec571.